components/brave_wallet_ui/page/screens/send/components/account-selector/account-selector.tsx

```diff
@@ -109,6 +126,7 @@ export const AccountSelector = (props: Props) => {
               isSelected={
                 account.accountId.uniqueKey === selectedAccountId?.uniqueKey
               }
+              accountAlias={fvmTranslatedAddresses?.[account.accountId.address] || ''}
```

Comment:
No need to fallback to `''` since `accountAlias` can also be `undefined` as per the props definition.